### ISSUE TITLE
Improve support for accessibility

### ISSUE THREAD

**Issue by samreid** [phetsims/forces-and-motion-basics#110]
Jonathan Hung pointed out a number of issues that could be improved for accessibility.  Here is the original report, from Nov 13

Hi everyone,

Anastasia and I did a quick keyboard accessibility walkthrough of the latest tug-of-war game in the "Forces and Motion Basics" PHeT simulation, with the developer "accessibility flag" enabled.

* Link to github version (the version we tested)
* Link to production version on the PHeT website 

This is what we discovered. NOTE: we restricted the scope to keyboard interaction only at this time.

Summary:
Priority should be given to  giving keyboard access to items that are currently not keyboard accessible (i.e. reset button etc.). This will at least make the simulation completely navigable by keyboard. 

Once this is done, attention can then be given to tweaking the interaction to make the keyboard experience more pleasant. 

Some features not keyboard focusable:
Some buttons on the interface are mouse-clickable, but not focusable by keyboard.
- [x] reset simulation button
- [x] audio / mute button
- [x] about item in popped-up menu
- [x] return button

"Go" button tab order:
Consider changing the tab order to allow a player to get the sim working as quickly as possible. Currently the tab order is:

1. blue players, left to right
2. red players, left to right
3. sum of forces checkbox option
4. display values checkbox option 
5. go button
6. menu button (lower right corner)

- [x] The "Go" button can be moved up to 3rd in the tab order.

Other interaction notes:

- [x] Activating the menu button makes screen dim and the “about" popup appears, but you can't dismiss the popup by pressing Space again.

- [x] If all 4 players from a team are on the rope, attempting to remove a team member just subtracts their force from the arrow, but they are visually still on the rope. It's possible to get all pullers off the rope, but it requires focusing on other items first and then back on the pullers.

- [x] Go button will get focus and can be activated even if no players have been moved to rope.

- [x] Reset (clicking it, since it doesn’t get keyboard focus) doesn't remove all players from the rope (visual bug only; sum of forces is reset to zero)

- [ ] Return and circular reset button are ambiguous to someone who has not tried the simulation.

- [x] With keyboard interaction, pullers are placed into predetermined spots on the rope when activated. Mouse interaction is more flexible allowing the user to place pullers on any spot. Keyboard users should get the same freedom to choose where to place their pullers.

Let us know of any comments or questions.

Cheers,
- Jon.

**Comment by ariel-phet** [comment]
Closing this issue since FAMB and keyboard nav are an ongoing effort to which @samreid is regularly contributing.

**Comment by samreid** [comment]
By the way, the NVDA problem mentioned above has a solution well known by NVDA users.  Called something like "application mode" it enables space+enter to behave like normal.  Learned about this through Colin, and email threads should elaborate if necessary.
